lib/rmt/mirror.rb

```diff
@@ -135,9 +158,15 @@ def mirror_data(primary_files, deltainfo_files)
     @downloader.destination_dir = @repository_dir
     @downloader.cache_dir = nil
 
-    package_files =
+    package_references =
       parse_mirror_data_files(deltainfo_files, RepomdParser::DeltainfoXmlParser) +
       parse_mirror_data_files(primary_files, RepomdParser::PrimaryXmlParser)
+
+    package_files = package_references.map do |reference|
```

Comment:
`package_files` is still made up of `FileReference` instances. :man_shrugging:

Reply:
I tried to be more explicit in bbbe807. What do you think?